### PR TITLE
Fork updates, new test comparison method

### PR DESCRIPTION
**Pull request by builtbywill**
* Pull in commits from upstream
* Do not use symbolic equivalence for unit tests (with some exceptions)

### CODE REVIEW

File: latex2sympy.py [hunk 1]
```diff
@@ -717,3 +709,34 @@ def get_differential_var_str(text):
...
+    def __new__(cls, *args, in_parsing=False, **options):
+        if in_parsing:
+            args = (args[0], sympy.Pow(args[1], -1, evaluate=False))
+        return super().__new__(Div, *args, **options)
+
+    def _sympyrepr(self, expr, order=None):
+        return "Div(%s)" % ",".join(map(sympy.srepr, self.args))
+
+
+class Root(sympy.Pow):
```
Comment (tragessere) [inline latex2sympy.py]: I feel like this class should be doing the reverse power itself, it doesn't really make sense to have it behave the same as `Pow` when it's supposed to be the opposite.

File: latex2sympy.py [hunk 2]
```diff
@@ -301,7 +290,7 @@ def convert_postfix(postfix):
             if ev.eval_at_sub():
                 at_a = do_subs(exp, ev.eval_at_sub())
             if at_b is not None and at_a is not None:
-                exp = sympy.Add(at_b, -1 * at_a, evaluate=False)
+                exp = Sub(at_b, at_a, evaluate=False)
```
Comment (tragessere) [inline latex2sympy.py]: Without the `in_parsing` argument I think this is behaving the same as `Add`. Not sure if that should need to be passed in though.

File: latex2sympy.py (hunk 1 above)
Comment (builtbywill) [reply latex2sympy.py]: @tragessere i haven't tested it, but im pretty sure this was to have a non-evaluated exp for sqrt and root, while also giving a custom output for `srepr`

File: latex2sympy.py (hunk 2 above)
Comment (builtbywill) [reply latex2sympy.py]: @tragessere i think you are right, i will try changing it

File: latex2sympy.py [hunk 3]
```diff
@@ -717,3 +709,34 @@ def get_differential_var_str(text):
     if text[0] == "\\":
         text = text[1:]
     return text
+
+
+class Div(sympy.Mul):
+
+    def __new__(cls, *args, in_parsing=False, **options):
```
Comment (tragessere) [inline latex2sympy.py]: Actually I'm not really liking the `in_parsing` argument for any of these new classes. I'm not seeing why we would ever want them to do the opposite of what they say they do (and by default).

Comment (builtbywill) [reply latex2sympy.py]: @tragessere 🤔 hm, good point

**Comment by builtbywill** [comment]
@tragessere updated

**Pull request merged**